operator/cmd/mesh/profile-common.go

```diff
@@ -50,13 +50,24 @@ func genIOPS(inFilename []string, profile, setOverlayYAML, ver string, force boo
 		return "", nil, fmt.Errorf("could not Unmarshal overlay Set%s: %s", setOverlayYAML, err)
 	}
 	if inFilename != nil {
-		b, err := ReadLayeredYAMLs(inFilename)
+		inputYaml, err := ReadLayeredYAMLs(inFilename)
```

Comment:
this approach still has some issues - now we are outputting console info in a function which is supposed to be generating an IOPs and also overloading the meanings of what inFilename can contain. if possible, it would be cleaner to do the translation earlier so that by the time we reach this function we are already passing in the correct format.

Reply:
> this approach still has some issues - now we are outputting console info in a function which is supposed to be generating an IOPs and also overloading the meanings of what inFilename can contain. if possible, it would be cleaner to do the translation earlier so that by the time we reach this function we are already passing in the correct format.

This function also read baseYAML from a downloaded tmp dir for a given version:
```
pkgPath, err := fetchInstallPackage(helm.InstallURLFromVersion(ver))
```
This will also read an ICPS rather than IOP.

Reply:
Since the baseYAML is not passed from outside, and the func here has subtle ordering, I'd like to not move it for now.